src/validation.h

```diff
@@ -485,6 +486,10 @@ class CoinsViews {
     //! can fit per the dbcache setting.
     std::unique_ptr<CCoinsViewCache> m_cacheview GUARDED_BY(cs_main);
 
+    //! Used as an empty view that is only passed into ConnectBlock to help speed up block validation,
+    //! as well as not pollute the underlying cache with newly created coins in case the block is invalid.
```

Comment:
It is not clear from this description why this data structure needs to be persisted, if it only holds ephemeral data. I think it should be mentioned that it owns handles to worker threads.